web/e2e/tests/routes/Landing.e2e.js

```diff
@@ -0,0 +1,33 @@
+// @flow
```

Comment:
I think it should be lowercase: `landing.e2e.js`

Reply:
I think this is fine for now as this task is only about the setup.